molecule/testinfra/staging/common/test_grsecurity.py

```diff
@@ -160,8 +143,7 @@ def test_apt_autoremove(Command):
     assert "The following packages will be REMOVED" not in c.stdout
 
 
-@pytest.mark.skipif(os.environ.get('FPF_GRSEC', 'true') == "false",
-                    reason="Need to skip in environment w/o grsec")
+@pytest.mark.xfail(strict=True)
```

Comment:
Can you comment the reason for `xfail`-ing this instead of skipping? And if possible, can you narrow the scope of the `xfail` to the exception we expect to see?